apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector-control.js

```diff
@@ -28,6 +32,8 @@ export const TemplateSelectorControl = ( {
 	onTemplateSelect = noop,
 	onTemplateFocus = noop,
 	siteInformation = {},
+	currentTemplate = '',
```

Comment:
minor: side question: what do you think to set up a default `currentTemplate` with `blank`? Just an idea.